features/accesskey.yml

```
@@ -0,0 +1,9 @@
+name: AccessKey
+description: The `accessKey` global attribute gives a hint for generating a keyboard shortcut for the current element. The attribute value must consist of a single printable character.
```

For all of these, I think we should strive to be clear that it's an HTML attribute:

```suggestion
description: The `accessKey` global HTML attribute gives a hint for generating a keyboard shortcut for the current element. The attribute value must consist of a single printable character.
```